confignode/src/main/java/org/apache/iotdb/confignode/manager/node/NodeManager.java

```diff
@@ -261,6 +263,18 @@ public DataSet registerDataNode(RegisterDataNodePlan registerDataNodePlan) {
     dataSet.setDataNodeId(
         registerDataNodePlan.getDataNodeConfiguration().getLocation().getDataNodeId());
     dataSet.setConfigNodeList(getRegisteredConfigNodes());
+    return dataSet;
+  }
+
+  /**
+   * Get configuration
+   *
+   * @return DataNodeConfigurationDataSet. The TSStatus will be set to SUCCESS_STATUS when register
```

Comment:
`The TSStatus will be set to SUCCESS_STATUS when register success, and DATANODE_ALREADY_REGISTERED when the DataNode is already exist.`  
This comment is described for method `registerDataNode`, method `getConfiguration` doesn't need it.